src/server/multi_command_squasher.cc

```diff
@@ -112,6 +112,8 @@ MultiCommandSquasher::SquashResult MultiCommandSquasher::TrySquash(StoredCmd* cm
 
   cmd->Fill(&tmp_keylist_);
   auto args = absl::MakeSpan(tmp_keylist_);
+  if (args.size() == 0)
```

Comment:
Curious, didn't `DetermineKeys` below handle this case or ?


Also general small nits (I do not care if you apply this or not :smile: )

1. `span` contains `empty()` 
2. We can also use `NumArgs()` and avoid the two calls above.

Reply:
Switched to use `empty()`, thanks!
Re/ `NumArgs()`, that's a feature of the command, not the args